apache/superset
# feat: add empty states to sqlab editor and select

### SUMMARY
This pr creates an empty state for the dropdown for the table selector and query editor when there is no database selected.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
after

https://user-images.githubusercontent.com/17326228/162277445-cefc2bdf-6ee5-4ce4-912d-dafb85dda4fc.mov


### TESTING INSTRUCTIONS

Go to sql editor. If there is no db selected to query there should be an empty state for the query editor. If there is no db connected to db select dropdown should show empty state with link to foward user to db listview.

### ADDITIONAL INFORMATION
